tests/pipeline_test.py

```diff
@@ -492,16 +491,27 @@ class A(sl.Scope[T], int):
     assert pl.compute(A[str]) == A(2)
 
 
-def test_building_graph_with_loop_raises_CycleError() -> None:
+def test_building_graph_with_cycle_succeeds() -> None:
     def f(x: int) -> float:
         return float(x)
 
     def g(x: float) -> int:
         return int(x)
 
     pipeline = sl.Pipeline([f, g])
-    with pytest.raises(CycleError):
-        pipeline.build(int)
+    _ = pipeline.build(int)
+
+
+def test_computing_graph_with_cycle_raises_CycleError() -> None:
+    def f(x: int) -> float:
+        return float(x)
+
+    def g(x: float) -> int:
+        return int(x)
+
+    pipeline = sl.Pipeline([f, g])
+    with pytest.raises(sl.scheduler.CycleError):
+        pipeline.compute(int)
```

Comment:
Just a question,
Will it use `DaskScheduler` in `tox -e dask` and `NaiveScheduler` in `tox -e py38`...?

Reply:
Yes.